modules/util/source-formatter/README.markdown

```diff
@@ -100,7 +100,7 @@ Checks are configured in the following files:
 - ### By File Extensions:
    - [.action, .function, .jelly, .jrxml, .macro, .pom, .project, .properties, .svg, .testcase, .toggle, .tpl, .wsdl, .xml or .xsd](src/main/resources/documentation/xml_source_processor_checks.markdown#checks-for-action-function-jelly-jrxml-macro-pom-project-properties-svg-testcase-toggle-tpl-wsdl-xml-or-xsd)
    - [.bnd](src/main/resources/documentation/bnd_source_processor_checks.markdown#checks-for-bnd)
-   - [.bnd, .ftl, .gradle, .java, .jsp, .jspf, .scss or .vm](src/main/resources/documentation/upgrade_source_processor_checks.markdown#checks-for-bnd-ftl-gradle-java-jsp-jspf-scss-or-vm)
```

Comment:
@MicaelleSilvaa , the documentation is wrong, please redo it.

Reply:
resolved